projects/angular-image-viewer/src/lib/angular-image-viewer.component.ts

```diff
@@ -190,8 +180,6 @@ export class AngularImageViewerComponent implements OnInit, OnChanges {
   reset() {
     this.scale = 1;
     this.rotation = 0;
-    this.translateX = 0;
```

Comment:
bec of this, when we drag somewhere and move on to next image then its not coming in center.